lib/Dialect/AIE/IR/AIEDialect.cpp

```diff
@@ -1096,6 +1093,19 @@ int SwitchboxOp::getNumSourceConnections(WireBundle bundle) {
     default:
       return 0;
     }
+  else if (getTileOp().isMemTile())
+    switch (bundle) {
+    case WireBundle::DMA:
+      return 6;
+    case WireBundle::North:
+      return 4;
+    case WireBundle::South:
+      return 6;
+    case WireBundle::Trace:
```

Comment:
I thought I had plumbed this through the targetmodel already? Oops, guess I never committed it!